src/study-setup.js

```diff
@@ -0,0 +1,118 @@
+// TODO:
+
+// - Replace survey URLs
+// - Figure out how to target English only, United States and Canada
+// - Exclude users with containers extensions to avoid awesome bar conflicts
+// - Ping telemetry on start & on end
+// - Ping on every page load that is an article
+// - Need to handle end if occured through
+//    - navigate away (beforeunload in contentscript ?)
+//    - tab closed (beforeunload in content script ?)
+//    - close window
+//    - quit firefox
```

Comment:
Are these all still to-do?

I'm assuming that the only surveying this should do is through the popup, and there won't be an end-of-study survey.